api-test/bdd/steps.go

```
@@ -0,0 +1,596 @@
+	s := getState(ctx)
+	passed := s.lastStatus == code
+	s.recordAssertion("HTTP status", strconv.Itoa(code), strconv.Itoa(s.lastStatus), passed)
+	if !passed {
+		return fmt.Errorf("expected HTTP %d, got %d (body: %s)", code, s.lastStatus, snippet(s.lastBody))
+	}
+	return nil
+}
+
+func theResponseStatusShouldBeClass(ctx context.Context, class string) error {
+	s := getState(ctx)
+	ok := false
```

golangci-lint (2.12.2)

[error] assigned to ok, but reassigned without using the value

(wastedassign)